src/runtime_src/core/pcie/emulation/cpu_em/generic_pcie_hal2/shim.h

```diff
@@ -174,7 +219,7 @@ namespace xclcpuemhal2 {
       int xclRegRW(bool rd, uint32_t cu_index, uint32_t offset, uint32_t *datap);
       int xclRegRead(uint32_t cu_index, uint32_t offset, uint32_t *datap);
       int xclRegWrite(uint32_t cu_index, uint32_t offset, uint32_t data);
-
+      std::vector<std::string> parsedMsgs;
```

Comment:
Where is this parsedMsgs is used, I think you have removed the local parseLog API which is using this. So you can remove this even.

Reply:
removed this declartion too